french/dessine/porte/winnie/winnie.md

```diff
@@ -0,0 +1,14 @@
+Alors que vous perdiez tout espoir, vous entendez un "WOOSH" derrière vous. 
+En vous retournant, vous vous retrouvez nez-à-nez avec un ourson en peluche
+à l'air jovial.
+Il est entièrement jaune à l'exception d'un t-shirt rouge et fait approximativement la taille d'un petit enfant.
+Il s'adresse à vous d'une voix douce :
+"J'ai entendu tes pleurs et tes cris et j'ai décidé de venir à ton secours. Je vois que
+tu es en fort mauvaise posture. Ne bouge pas, je vais te dégager de là."
+
+Aussitôt il se met à lapper goulûment le miel qui recouvre la porte, les murs et vos mains.
+En quelques minutes, vous êtes libre ! Vite, vous éloignez vos mains avant que le miel qui continue à couler ne vous emprisonne à nouveau.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.